conf/default-vars.yaml

```diff
@@ -48,7 +48,10 @@ vars:
 
   border:
     size: 1px
-    radius: 3px
+    radius:
+      default: 3px
+      lg: 5px
```

Comment:
any reason why?  Don't you want your border-radius to be consistent across a project?